docs/content/preview/integrations/camunda.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Camunda
+linkTitle: Camunda
+description: Use Camunda with YSQL
+aliases:
+section: INTEGRATIONS
+menu:
+  preview:
+    identifier: camunda
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+[Camunda](https://camunda.com/) is a Java-based framework supporting BPMN (Business Process Modeling Notation) for workflow and process automation, CMMN (Case Management Model and Notation) for Case Management and DMN (Decision Model and Notation) for business decision management.
+
+Camunda supports PostgreSQL as a datasource. Because YugabyteDB is wire compatible with PostgreSQL, it works out of the box with Camunda as a datasource. This document covers how to:
+
+- Configure Camunda to use YugabyteDB (YSQL) as a data source.
+- Run a small hello world example in Camunda.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you're new to YugabyteDB, you can download, install, and have YugabyteDB up and running in minutes by following the steps in [Quick start](/preview/quick-start/). Alternatively, you can use [YugabyteDB Managed](http://cloud.yugabyte.com/) to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+- Java JDK 1.8 or later is installed.
+- NodeJS v10 or later is installed.
+- Camunda Platform 7 and Modeler are installed following the instructions on Camunda's [quick-start page](https://docs.camunda.org/get-started/quick-start/install/).
+
+## Configure Camunda Platform 7
+
+1. Locate the `spring.datasource` section in the following files:
+
+    - `camunda-bpm-run-7.17.0/configuration/default.yml`
+    - `camunda-bpm-run-7.17.0/configuration/production.yml`
+
+1. Modify the `spring.datasource` section that uses standard PostgreSQL JDBC driver. Change the `url` value to point to the YugabyteDB cluster you started:
+
+    ```yaml
+    # datasource configuration is required
+    spring.datasource:
+      url: jdbc:postgresql://localhost:5433/yugabyte
+      driver-class-name: org.postgresql.Driver
+      username: yugabyte
+      password: yugabyte
+    ```
+
+1. Download the PostgreSQL JDBC driver [JAR file](https://jdbc.postgresql.org/download/postgresql-42.3.5.jar) and place it in the `camunda-bpm-run-7.17.0/configuration/userlib` directory.
+
+Alternatively, you can also use YugabyteDB JDBC Driver to connect Camunda to a YugabyteDB cluster.
+To do so, modify the `spring.datasource` section  in each file to point to the YugabyteDB cluster you started, and replace the value of `driver-class-name` with `com.yugabyte.Driver`.
```

Comment:
Fixing formatting... But also, do we really want to present our driver as an alternative, rather than the default choice?

```suggestion
    <br/><br/>Alternatively, you can also use the YugabyteDB JDBC driver to connect Camunda to a YugabyteDB cluster. To do so, modify the `spring.datasource` section  in each file to point to the YugabyteDB cluster you started, and replace the value of `driver-class-name` with `com.yugabyte.Driver`.
```

Reply:
The idea is to present that `YugabyteDB` can be used with the `Camunda` using the `Postgres driver`.
If any user wishes to use the `YugabyteDB driver` instead of the `Postgres driver`, then it can also be done.****